index.html

```diff
@@ -83,6 +83,204 @@ <h1>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+          </div>
+
+           <!-- TONIC2 -->
+        
+        <div class="flex">
+            <div class="work-card">
+              <img src="img/IMG 2.png" alt="Tonic" />
+              <div class="work-card1">
+                <h2>Tonic</h2>
+                <div class="canopy">
+                  <span>CANOPY</span>
+                  <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+                <p class="daily">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul class="lang">
+                  <li class="lang-item">html</li>
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+          </div>
+
+           <!-- MULTI-POST STORIES2 -->
+        
+        <div class="flex">
+            <div class="work-card">
+              <img src="img/IMG 4.png" alt="Tonic" />
+              <div class="work-card1">
+                <h2>Multi-Post Stories</h2>
+                <div class="canopy">
+                  <span>CANOPY</span>
+                  <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+                <p class="d-text">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul class="lang">
+                  <li class="lang-item">html</li>
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+        </div>
+
+
+        <div class="main-container">
+            <div class="introduction">
+              <h1>About Myself</h1>
+              <p>
+                I’m a software developer! I can help you build a product, feature
+                or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t
+                hestiate to contact me.
+              </p>
+            </div>
+            <div class="connect">
+              <a href="#">LET’S CONNECT</a>
+              <div class="intro-card2">
+                <ul class="icon">
+                  <li>
+                    <a href="#"> <img src="img/Vector.svg" /> </a>
+                  </li>
+                  <li>
+                    <a href="#">
```

Comment:
- [ ] Kindly ensure there is a spacing between the `work section` and the `about me section` like in the Figma design
- [ ] Work on the positioning and spacing of elements in your `about me section` to match the original design.

![image](https://user-images.githubusercontent.com/98453979/219644287-fb840cff-d585-45b1-92c8-e022b2610014.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/219644746-0ef3de4c-a7ff-4ee5-94e0-fdb6fca1003c.png)